caddyhttp/fastcgi/fastcgi_test.go

```diff
@@ -293,4 +296,24 @@ func TestBuildEnv(t *testing.T) {
 	envExpected["CUSTOM_URI"] = "custom_uri/fgci_test.php?test=blabla"
 	envExpected["CUSTOM_QUERY"] = "custom=true&test=blabla"
 	testBuildEnv(r, rule, fpath, envExpected)
+
+	// 6. Test Caddy-Rewrite-Original-URI header is not removed
+	const internalRewriteFieldName = "Caddy-Rewrite-Original-URI"
```

Comment:
Caddy-Rewrite-Original-URI is also set as a constant in [rewrite.go](https://github.com/mholt/caddy/blob/bedad34b25cfcc67620fa5b854fed020547efbdf/caddyhttp/rewrite/rewrite.go#L244) using the const name  `headerFieldName`

`internalRewriteFieldName` in fastcgi.go with `headerFieldName` and everything compiles and works, but he test fail as headerFieldName is undefined.  I still get an error even if I add rewrite.go to the import block.

2 Questions
 
  1. Should we use `internalRewriteFieldName` or `headerFieldName`  as the const or create a new const name 
  2. Where should I make the declaration to work in all tests?